_extensions/bluesky-comments/bluesky-comments.js

```diff
@@ -454,48 +437,35 @@ class BlueskyComments extends HTMLElement {
     });
   }
 
-  #renderWarning(labels) {
-    if (!labels?.length) return;
+  #renderWarning(post) {
+    const { labels } = post;
+
+    if (!labels?.length) return '';
 
-    const formattedLabels = labels.map(l => ({
-      value: l.val.charAt(0).toUpperCase() + l.val.slice(1),
-    }));
+    const labelDisplay = {
+      sexual: 'adult content',
+    };
+
+    // TODO: Filter out negated labels, see https://atproto.blue/en/latest/atproto/atproto_client.models.com.atproto.label.defs.html
```

Comment:
Should this be a new issue ticket?

Reply:
Great point! New issue here: https://github.com/quarto-ext/bluesky-comments/issues/36